Library/Homebrew/system_config.rb

```diff
@@ -85,6 +85,19 @@ def describe_ruby
       end
     end
 
+    def describe_system_ruby
+      s = ""
+      case RUBY_VERSION
+      when /^1\.[89]/, /^2\.0/
```

Comment:
What do you suggest to remove duplication? A new method like `describe_homebrew_ruby_version`?